scripts/php/process.php

```diff
@@ -119,7 +119,24 @@ private function buildMessageText () {
         // what is the submitter's email address? Who submitted the form?
         case 'email':
           $value = $this->cleanse($value);
-          $this->from = filter_var($value,FILTER_SANITIZE_EMAIL);
+          if($this->validateEmail($value)){
+             $this->from = filter_var($value,FILTER_SANITIZE_EMAIL);
+          } else {
+            if($_POST['recipient']=='libraries-tellus@mit.edu'){
+              // Form came from Tell Us, which allows anonymous submissions
+              $this->log->write('[Form] Email - Sender address verification failed, using stand-in address for Tell Us');
+              $this->log->write("Anonymous submission, using stand-in email address");
+              $strFrom = "tellus-lib@mit.edu";
+            } else {
+              // Flag as invalid
+              $this->log->write('[Form] Email - Sender address verification failed, rejecting submission');
+              $this->log->write("[Form] Email validation failed");
+              // $valid = FALSE;
```

Comment:
Seeing this reminds me that I'd like us to be able to get to a better stand-in email address than my own (which is set up on L25) - @frrrances do you have a recommended address we could use?